GPflowOpt/acquisition/acquisition.py

```diff
@@ -153,32 +165,38 @@ def constraint_indices(self):
     def objective_indices(self):
         """
         Method returning the indices of the model outputs which are objective functions.
+        
         By default all outputs are objectives
+        
+        :return: int ndarray, M
         """
         return np.setdiff1d(np.arange(self.data[1].shape[1]), self.constraint_indices())
 
     def feasible_data_index(self):
         """
         Returns a boolean array indicating which data points are considered feasible (according to the acquisition
         function(s) ) and which not.
+        
         By default all data is considered feasible
+        
         :return: boolean ndarray, N
         """
         return np.ones(self.data[0].shape[0], dtype=bool)
 
     def setup(self):
         """
-        Method triggered after calling set_data().
-
-        Override for pre-calculation of quantities used later in
```

Comment:
hmm, this could indeed be turned into private as it is now called from set_data. But I don't think it is very nice to present an interface for users to implement where it is a common practice to override a protected method.